src/Native/TestContext.js

```diff
@@ -121,7 +121,8 @@ var _user$project$Native_TestContext = (function () { // eslint-disable-line no-
         init: app.init,
         update: program.elmTestable.update,
         subscriptions: program.elmTestable.subscriptions,
-        view: program.elmTestable.view
+        view: program.elmTestable.view,
+        locationToMessage: program.elmTestable.locationToMessage || { ctor: 'Nothing' }
```

Comment:
Do you think it's best to put the fallback Nothing here, or should we instead update the Html.program intercept to add it there?

Reply:
Hmmm I like the fallback because it is less native code from other parts to intercept (in this case, Html.program)

And what if there are other programs we don't know of? idk